app/components/c-mini-cart.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div :class="b({ active })" @click="onClick">
+    <div v-if="miniCartHasItems" :class="b('count')">{{ miniCart.cartQuantity }}</div>
+    <div :class="b('icon')"><e-icon inline icon="i-cart"/></div>
+    <div v-if="miniCartHasPrice" :class="b('total')">{{ displayPrice }}</div>
+  </div>
+</template>
+
+<script>
+  import { mapGetters } from 'vuex';
+
+  export default {
+    name: 'c-mini-cart',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Forces the active state
+       */
+      active: {
+        type: Boolean,
+        default: false,
+      },
+    },
+    // data() {
+    //   return {};
+    // },
+
+    computed: {
+      ...mapGetters('cart', [
+        'miniCart',
+        'miniCartHasItems',
+        'miniCartHasPrice',
+      ]),
+      displayPrice() {
+        return this.$n(this.miniCart.netTotal / 100, 'currency', 'de-CH'); // TODO - remove hardcoded locale
+      },
+    },
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      onClick(event) {
+        /**
+         * Click event.
+         *
+         * @event click
+         * @type {object}
+         */
+        this.$emit('click', event);
+      },
+    },
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  .c-mini-cart {
+    position: relative;
+    padding: 10px;
+    cursor: pointer;
+
+    @include media(sm) {
+      padding: 0;
+    }
+
+    &__icon {
+      width: 22px;
+      height: 24px;
+
+      @include media(sm) {
+        margin: 15px auto 0 auto;
+      }
+
+      path {
+        fill: $color-grayscale--1000;
+      }
+    }
+
+    &--active &__icon,
+    &:active &__icon,
+    &:hover &__icon {
+      path {
+        fill: $color-primary--1;
+      }
+    }
+
+    &__count {
+      @include font($font-size--10, 10px, $font-weight--regular);
+
+      position: absolute;
+      right: 50%;
```

Comment:
- [ ] I would recommend to align this on the left and use `transformation: translateX(-50%)` to center it on a specific point. So when the number increases the element extends to left an right and not only left. I think this would give us a bit more flexible space before conflicting with other elements outside of this component.

Reply:
I tried but i am not sure how i am supposed to do this. I can't seem to position the counter on a specific point, especially because the total amount number changes the size of the container as well.